File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--simple-data-binding-in-vsto-add-in-project_2.cs
            this.adventureWorksDataSet = new AdventureWorksLTDataSet();
            this.customerTableAdapter = new AdventureWorksLTDataSetTableAdapters.CustomerTableAdapter();
            this.customerTableAdapter.Fill(this.adventureWorksDataSet.Customer);
            this.customerBindingSource = new System.Windows.Forms.BindingSource();

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--simple-data-binding-in-vsto-add-in-project_2.vb
        Me.adventureWorksDataSet = New AdventureWorksLTDataSet()
        Me.customerTableAdapter = New AdventureWorksLTDataSetTableAdapters.CustomerTableAdapter()
        Me.customerTableAdapter.Fill(Me.adventureWorksDataSet.Customer)
        Me.customerBindingSource = New System.Windows.Forms.BindingSource()